Description: Use curve25519-dalek instead of curve25519-dalek-ng
 The two are basically identical asides from "branding", and only
 the first one is in Debian.
Author: NoisyCoil <noisycoil@tutanota.com>
Forwarded: not-needed
Last-Update: 2024-07-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -50,7 +50,7 @@
 default-features = false
 [dev-dependencies.curve25519-dalek]
 version = "4"
-package = "curve25519-dalek-ng"
+features = ["rand_core"]
 
 [dev-dependencies.rand_chacha]
 version = "0.3"
